### PR TITLE
Migrate legacy ccdb complaint landing page to jinja

### PR DESCRIPTION
**Pull request by anselmbradford**
Migrate legacy ccdb complaint landing page to jinja. This page used the legacy base_update.html template.

## Removals

- Unused legacy CF and notification CSS.
- Remove unused CSS from landing-page.css

## Changes

- Move complaint-landing template from django to jinja and remove unused markup.
- Import DS CSS by reference.

## How to test this PR

1. Pull branch and run `gulp clean && gulp build` or equivalent.
2. Compare http://localhost:8000/data-research/consumer-complaints/ to https://www.consumerfinance.gov/data-research/consumer-complaints/
3. Page should be almost identical apart from minor differences:
 - Gap before "Things to know before…" well is larger.
 - Notification has moved from above the hero to below the "Things to know before…" well. (Inspect page and look for `m-notification`.)
 - Tilemap key is a gray instead of black text.
 - Tilemap state popup has larger text. (This and the above are a consequence of something coming in from cfpb-chart-builder and not worth diagnosing I think. The larger text seems more accessible.)

## Screenshots

Before:
Screen Shot 2021-03-25 at 1 17 20 PM

After:
Screen Shot 2021-03-25 at 1 19 05 PM

Before:
Screen Shot 2021-03-25 at 2 54 01 PM

After:
Screen Shot 2021-03-25 at 2 54 10 PM

Before:
Screen Shot 2021-03-25 at 2 54 34 PM

After:
Screen Shot 2021-03-25 at 2 54 30 PM

### CODE REVIEW

**Comment by anselmbradford** [comment]
Updated to move notification above well and convert well to true well:

Screen Shot 2021-03-29 at 1 41 14 PM

File: cfgov/cfgov/urls.py [hunk 1]
```diff
@@ -299,8 +298,9 @@ def empty_200_response(request, *args, **kwargs):
     ),
 
     re_path(
-        r'^data-research/consumer-complaints/$',
-        ComplaintLandingView.as_view(),
+        r'^data-research/consumer-complaints/$', TemplateView.as_view(
```
Comment (chosak) [inline cfgov/cfgov/urls.py:301]: Don't you still need to keep CLV around to leverage all of the various [contextual data](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L28-L39)? It's already configured to use [`complaint/complaint-landing.html`](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L26) so it should already use your new template.

Comment (anselmbradford) [reply cfgov/cfgov/urls.py]: Hmmmmm… you're right. Updated in https://github.com/cfpb/consumerfinance.gov/pull/6357/commits/bf6a3d50a44abc510459ffd2dacdb5fd06c7f94d

The template I migrated has three variables it uses to show notifications: `data_down`, `narratives_down`, and `technical_issues`. Do you see anywhere where the `ccdb_content_updates` [flag](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L36) is used? Can we delete that?

Comment (anselmbradford) [reply cfgov/cfgov/urls.py]: PR'd the flag removal [here](https://github.com/cfpb/consumerfinance.gov/pull/6359)

File: cfgov/jinja2/v1/complaint/complaint-landing.html [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+  </div>
+</div>
+<div class="block block__border-top block__padded-top block__flush-top">
+  <div class="section-five">
+    <h2 id="download-the-data">Download the data</h2>
+    <p class="download-text">Our reports highlight some of what we’re seeing in the complaints we handle, but know there are more ways to use, analyze and build on the data. That’s why we make it possible for you to download the data, once we have taken steps to remove personal information.</p>
+    <p class="download-text">You can download all complaint data as either a CSV or JSON file here, or you can download a subset of the data—such as all complaints for a specific product—by filtering the full data set and exporting your results.</p>
+    <ul class="m-list m-list__links">
+      <li class="m-list_item">
+        <a class="a-link a-link__icon" href="http://files.consumerfinance.gov/ccdb/complaints.csv.zip">
```
Comment (flacoman91) [inline cfgov/jinja2/v1/complaint/complaint-landing.html:247]: Any chance we can update this link to use **https:** protocol instead?  I'm getting some errors in chrome and it won't allow mixed protocols.

thanks

File: cfgov/jinja2/v1/complaint/complaint-landing.html [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+          <span class="a-link_text">Download all complaint data | CSV</span>
+        </a>
+      </li>
+      <li class="m-list_item">
+        <a class="a-link a-link__icon" href="http://files.consumerfinance.gov/ccdb/complaints.json.zip">
```
Comment (flacoman91) [inline cfgov/jinja2/v1/complaint/complaint-landing.html:252]: same with this one too

File: cfgov/jinja2/v1/complaint/complaint-landing.html (hunk 2 above)
Comment (anselmbradford) [reply cfgov/jinja2/v1/complaint/complaint-landing.html]: Changed in https://github.com/cfpb/consumerfinance.gov/pull/6357/commits/30aae2520a2af9a9c31c8f7330cc8f914f7d2aea

File: cfgov/jinja2/v1/complaint/complaint-landing.html (hunk 3 above)
Comment (anselmbradford) [reply cfgov/jinja2/v1/complaint/complaint-landing.html]: Changed in https://github.com/cfpb/consumerfinance.gov/pull/6357/commits/30aae2520a2af9a9c31c8f7330cc8f914f7d2aea

**Review by flacoman91 — APPROVED** [review]
Works well.  Awesome  🥇 
ie-map


Confirmed to work in IE11 too!

**Pull request merged**